Soomla/Assets/Plugins/Soomla/Core/Config/CoreSettings.cs

```diff
@@ -42,6 +43,9 @@ public class CoreSettings : ISoomlaSettings
 		static CoreSettings()
 		{
 			SoomlaEditorScript.addSettings(instance);
+
+			List<string> additionalDependFiles = new List<string>(); //Add files that not tracked in file_list
+			SoomlaEditorScript.addFileList("Core", "Assets/Soomla/core_file_list", additionalDependFiles.ToArray());
```

Comment:
@Blinkq, please make all keys in mFileList in lower-case, it should simplify to compare tags in different cases, when one of them is written correctly (in lower-case).